kategory/src/main/kotlin/kategory/data/Const.kt

```diff
@@ -0,0 +1,39 @@
+package kategory
+
+fun <A, T> ConstKind<A, T>.value(): A = this.ev().value
```

Comment:
I wonder if we can use Kotlin's delegation facilities to delegate all methods automatically in the codegen phase so `ConstKind` receives everything declared in `Const` and same for all other @higherkind data types.